cmd/gcp-controller-manager/node_annotator.go

```diff
@@ -143,6 +143,22 @@ func newNodeAnnotator(client clientset.Interface, nodeInformer coreinformers.Nod
 					return true
 				},
 			},
+			{
+				name: "resize-request-reconciler",
+				annotate: func(node *core.Node, instance *compute.Instance) bool {
+					resizeRequestName := extractResizeRequestLabel(instance)
```

Comment:
Can this metadata not be set directly as a label on the VM metadata? That way we do not need to add special logic to handle this here. Like any labels it would automatically be set.

Reply:
Unfortunately it can't - VMs within the same node pool can have a different value for this key.